Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:7.9.0-alpine
```

Comment:
not really, I'm working with this image for very log and nothing is wrong with 7.9 + there is npm >4
the only problem I has with installation only, it was connected to bower and we solved the issue with the special flag I introduced few days ago